test/cmdlineTests.sh

```diff
@@ -377,10 +379,13 @@ printTask "Running general commandline tests..."
             fi
         fi
 
-        scriptFiles="$(ls -1 "${tdir}/test."* 2> /dev/null || true)"
+        # jump into test dir
+        cd "${tdir}"
+
+        scriptFiles="$(ls -1 "./test."* 2> /dev/null || true)"
         scriptCount="$(echo "${scriptFiles}" | wc -w)"
 
-        inputFiles="$(ls -1 "${tdir}/input."* 2> /dev/null || true)"
+        inputFiles="$(ls -1 "./input."* 2> /dev/null || true)"
```

Comment:
I think we'd be better off without the `./` prefix for `input.*` and other files from the test dir. Their names are hard-coded, so there's no risk that we get one prefixed with `-`, which looks like an option name and this prefix does show through in some places. E.g. `model_checker_bmc_loop_iterations_no_argument/` output:

> ```
> Error: the argument ('./input.sol') for option '--model-checker-bmc-loop-iterations' is invalid
> ```

Though, TBH it's actually less of an issue than I expected - after going over all the files here I've seen very few of these. This is because we try hard to normalize the paths passed on the CLI.

Reply:
Ah yeah that is fair enough, let me fix that!